apps/condo/domains/ticket/schema/Ticket.js

```diff
@@ -304,6 +334,23 @@ const Ticket = new GQLListSchema('Ticket', {
                         }
                     }
                 }
+                const propertyChanged = resolvedData['property'] && resolvedData['property'] !== get(existingItem, 'property')
+                const propertyAddressChanged = resolvedData['propertyAddress'] && resolvedData['propertyAddress'] !== get(existingItem, 'propertyAddress')
+                const propertyAddressMetaChanged = resolvedData['propertyAddressMeta'] && resolvedData['propertyAddressMeta'] !== get(existingItem, 'propertyAddressMeta')
+                const newItem = { ...existingItem, ...resolvedData }
+                if (propertyChanged || propertyAddressChanged || propertyAddressMetaChanged) {
+                    const propertyId = get(newItem, 'property', null)
+                    const property = await getById('Property', propertyId)
```

Comment:
you also need to check access to that property. Or check the property.organisation.